### ISSUE TITLE
Licensing of @llamaindex/cloud npm package

### ISSUE THREAD

**Issue by peterwoodworth** [run-llama/LlamaIndexTS#983]
Hi folks,

We're interested in using the @llamaindex/cloud package [found on npm](https://www.npmjs.com/package/@llamaindex/cloud), however this package does not seem to contain licensing information. I assume that package comes from this repository, and is thus MIT licensed, but am not sure looking at the package if it maps to the code found in [this directory](https://github.com/run-llama/LlamaIndexTS/tree/main/packages/llamaindex/src/cloud).

tl;dr: Wondering where the source is for [@llamaindex/cloud npm package](https://github.com/run-llama/LlamaIndexTS/tree/main/packages/llamaindex/src/cloud), and if licensing information can be added to the distributed package so we know what rights we have when it comes to this software. Thanks!

**Comment by himself65** [comment]
Thanks for the tip; the license should be MIT because it's an API wrapper and utility library. 

For now it's private because it's very hacky to publish that, but I will move the package to this repo this week.

**Comment by himself65** [comment]
Please see @llamaindex/cloud@0.1.0

**Comment by peterwoodworth** [comment]
Thank you so much for the quick turnaround, very appreciated!
